Sources/Html/View.swift

```diff
@@ -27,3 +27,13 @@ extension View where M == [Node] {
     self.call = const([node])
   }
 }
+
+extension View where A == (), M == [Node] {
```

Comment:
P cool: these overloads let us omit the generic parameter from `View<()>` where we don't care about the data.